src/main/java/seedu/address/model/person/UserProfile.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.person;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import seedu.address.model.enumeration.FocusArea;
+import seedu.address.model.enumeration.Major;
+
+
+//@@author RomaRomama
+
+/**
+ * Presents profile of User
+ */
+
+public class UserProfile {
+
+    //Identity fields
+    private int year;
+    private int sem;
+    private Major major;
+    private FocusArea focusArea;
+
+    /**
+     * Every field must be present and not null
+     */
+    public UserProfile(int year, int sem, Major major, FocusArea focusArea) {
+        requireAllNonNull(year, sem, major, focusArea);
+        this.year = year;
+        this.sem = sem;
+        this.major = major;
+        this.focusArea = focusArea;
+    }
+
+    /**
+     *
+     * @return related infos
```

Comment:
If I'm not wrong, this javadoc would only apply for the method below. Might want to have a line between these two. The javadoc comment could be made into a general multi-line comment also since it won't be applying to any method in particular